### PR TITLE
Update Sourdough History

### PR DESCRIPTION
**Pull request by hendricius**
This adds another timeline for the sourdough bread history, adds more information and more references.

image

### CODE REVIEW

**Comment by hendricius** [comment]
@cedounet thanks ❤️ . What do you think of the additions here overall?

**Comment by cedounet** [comment]
On Tue, Dec 05, 2023 at 12:01:06PM +0000, Hendrik Kleinwaechter wrote:
> @cedounet thanks ❤️ . What do you think of the additions here overall?

I like it...  but I am somewhat OCD when math do not match :)  Takes me
a while when 10 cm means either 400 years or 12 000 years to be at ease
with it.

That aside it is a nice addition to the book, makes it well rounded
between tips, context and theory.


C/

File: book/history/sourdough-history.tex [hunk 1]
```diff
@@ -103,58 +111,154 @@ \chapter{The history of sourdough}%
 Quickly, similar processes were discovered when brewing beer
 or making wine. A small tiny batch of the previous production
 would be used for the next production. In this way, humans created
-modern bread yeasts, wine yeasts, and beer yeasts. Only in 1680,
-the scientist Anton van~Leeuwenhoek first studied yeast microorganisms
-under a microscope. Over time with each batch, the yeasts and bacteria
+modern bread yeasts, wine yeasts, and beer yeasts~\cite{egypt+beer}
```
Comment (cedounet) [inline book/history/sourdough-history.tex:114]: Missing a dot

File: book/history/sourdough-history.tex [hunk 2]
```diff
@@ -103,58 +111,154 @@ \chapter{The history of sourdough}%
...
+and not other chemical catalysts. He continued with his research and
+was the first person to isolate and grow pure yeast strains.
+Soon later in \num{1868} in the Fleischmann brothers Charles
+and Maximilian were the first to patent pure yeast strains
+for bread making. The yeasts offered
+were isolated from batches of sourdough. By \num{1879} the machinery was built
+to multiply the yeast in large centrifuges~\cite{fleischmann+history}.
 The pure yeast would prove to be excellent and turbocharged
-at leavening bread doughs. What would previously take 10~hours
+at leaving bread doughs. What would previously take 10~hours
```
Comment (cedounet) [inline book/history/sourdough-history.tex:218]: You sure ?

**Review by cedounet — CHANGES_REQUESTED** [review]

**Comment by cedounet** [comment]
image
Still some work to do but let me know if that is an option.

**Comment by hendricius** [comment]
@cedounet looks nice. What if you were to use the same read as the background of "Historic breadmaking"?

**Comment by cedounet** [comment]
On Fri, Dec 08, 2023 at 11:43:22AM +0000, Hendrik Kleinwaechter wrote:
> @cedounet looks nice. What if you were to use the same read as the
> background of "Historic breadmaking"?

I found it a bit too light for text... YMMV as usual.  Will fine tune
then we can discuss.

**Comment by cedounet** [comment]
Still need to deal with 12000 BC but as is for now
image


FYI here is the piknk version:
image

**Comment by cedounet** [comment]
Here is the latest, I will push code.
OK latest:
image

**Review by cedounet — APPROVED** [review]
LGTM with my changes... if you are OK with it, then hit merge.

**Comment by hendricius** [comment]
@cedounet thank you! Lovely improvements!

**Pull request merged**